extensions/sql-database-projects/src/dialogs/addSqlBindingQuickpick.ts

```diff
@@ -3,8 +3,11 @@ import { BindingType } from 'vscode-mssql';
 import * as constants from '../common/constants';
 import * as utils from '../common/utils';
 import { PackageHelper } from '../tools/packageHelper';
+import { TelemetryActions, TelemetryReporter, TelemetryViews } from '../common/telemetry';
 
 export async function launchAddSqlBindingQuickpick(uri: vscode.Uri | undefined, packageHelper: PackageHelper): Promise<void> {
+	TelemetryReporter.sendActionEvent(TelemetryViews.SqlBindingsQuickPick, TelemetryActions.sqlBindingsQuickPickStart);
```

Comment:
Having the action be `sqlBindingsQuickPickStart` is a little redundant since the view is already `sqlBindingsQuickPick`

Reply:
changed to startAddSqlBinding and the action to finishAddSqlBinding